feature-group-definitions/promise-any.yml

```diff
@@ -1,5 +1,5 @@
 name: Promise.any()
-description: Promise.any() static method
+description: The `Promise.any()` static method TODO.
```

Comment:
```suggestion
description: The `Promise.any()` static method returns a promise that fulfills as soon as the first of an iterable of promises fulfills, with that promise's value. Otherwise, it rejects with an `AggregateError` when all of the promises have rejected.
```